src/fetchapi/http.lib.js

```diff
@@ -0,0 +1,59 @@
+class SmallHTTP {

```

Comment:
Why don't export just object with fields `{get: () =>..`? - I don't see benefits from using of class here

Reply:
Will try this way